### PR TITLE
Suppot to change close button icon

### PR DESCRIPTION
**Pull request by baskvava**
Add close props for passing custom close component from outside of the component

### CODE REVIEW

File: src/rodal.js [hunk 1]
```diff
@@ -79,6 +87,7 @@ class Rodal extends React.Component {
     measure: 'px',
     visible: false,
     showMask: true,
+    close: null,
```
Comment (chenjiahan) [inline src/rodal.js]: How about rename to `closeIcon`? same as [ant-design](https://ant.design/components/modal)

File: src/rodal.js [hunk 2]
```diff
@@ -58,6 +65,7 @@ class Rodal extends React.Component {
     measure: PropTypes.string,
     visible: PropTypes.bool,
     showMask: PropTypes.bool,
+    close: PropTypes.elementType,
```
Comment (chenjiahan) [inline src/rodal.js]: Please add the new prop to document

File: src/rodal.js (hunk 1 above)
Comment (baskvava) [reply src/rodal.js]: Sounds good. It's much clear than just naming close

File: README.md [hunk 3]
```diff
@@ -52,26 +52,27 @@ class App extends React.Component {
...
+| Property         | Type       | Default | Description                                          |
+| ---------------- | ---------  | ------- | ---------------------------------------------------- |
+| width            | number     | 400     | width of dialog                                      |
+| height           | number     | 240     | height of dialog                                     |
+| measure          | string     | px      | measure of width and height                          |
+| onClose          | func       | /       | handler called onClose of modal                      |
+| onAnimationEnd   | func       | /       | handler called onEnd of animation                    |
+| visible          | bool       | false   | whether to show dialog                               |
+| showMask         | bool       | true    | whether to show mask                                 |
+| closeIcon        | React.Node | /       | custom close icon component
```
Comment (baskvava) [inline README.md:64]: Add closeIcon to doc

**Pull request closed without merging**